apps/sms/views/conversation/test/unit/recipients_test.js

```diff
@@ -143,6 +143,24 @@ suite('Recipients', function() {
       assertDeepEqual(recipient, fixture);
     });
 
+    test('recipients.add() parenthesis ', function(done) {
+        recipients.on('add', function(count, added) {
```

Comment:
__issue:__ Mmm can't you just use the same approach as in the test below (without ```done```, subscribing for the ```add``` event and using ```isValid``` method)? Moreover test doesn't work as expected, as you call ```done``` for the first number and the second one is not even checked :)

So what about improving ```isValid``` method above by adding additional checks for the ```candidate.isQuestionable``` and ```candidate.isValid``` there as well?